server/controllers/finance/transferts.js

```diff
@@ -0,0 +1,145 @@
+/**
+* The /transferts HTTP API endpoint
+*
+* @module finance/transferts
+*
+* @desc This module is responsible for handling all crud operations relatives
+* to financial transferts and define all transferts api functions
+*
+* @required lib/db
+* @required lib/guid
+* @required journal/primarycash
+*/
+
+'use strict';
+
+var db              = require('../../lib/db'),
+    uuid            = require('../../lib/guid'),
+    journalTransfer = require('./journal/primarycash').transfer;
+
+/** get the list of all transferts */
+exports.list = getList;
+
+/** get the detail of a particular transfert */
+exports.detail = getDetail;
+
+/** create a new transfert record in the database */
+exports.create = create;
+
+/**
+* GET /transferts
+* GET /transferts?limit={number}
+*
+* @desc This function is responsible for getting transferts from the database
+*/
+function getList(req, res, next) {
+  var sql,
+      qLimit = req.query.limit;
+
+  /** Getting the transfert module id */
+  sql = 'SELECT pcm.id FROM primary_cash_module pcm WHERE pcm.text = "transfert" ';
+
+  db.exec(sql)
+  .then(function (row) {
+    /** Getting transfert records */
+    sql =
+        'SELECT pc.uuid, pc.project_id, pc.type, pc.date, pc.currency_id, pc.account_id, pc.cost, pc.description, ' +
+        'pc.cash_box_id, pc.origin_id, pc.user_id, pci.document_uuid ' +
+        'FROM primary_cash pc ' +
+        'JOIN primary_cash_item pci ON pci.primary_cash_uuid = pc.uuid ';
+
+    qLimit  = Number(qLimit);
+
+    sql += 'WHERE pc.origin_id = ? ORDER BY pc.date DESC ';
+    sql += (qLimit) ? 'LIMIT ' + Math.floor(qLimit) : '';
+
+    return db.exec(sql, [row[0].id]);
```

Comment:
You should check that `row.length > 0`.  In this case, if a `pcm` without the text "transfert", you might send back a 500 Error, describing the problem.